src/fragments/lib/graphqlapi/js/subscribe-data.mdx

```diff
@@ -39,6 +38,27 @@ Amplify.configure({
 });
 ```
 
+### Subscription connection status updates
+
+While there are active subscription, the device is running your application and the network is healthy, your application will maintain a connection with AppSync. If you need to monitor the connection state, these states are made available via Hub.
```

Comment:
The first sentence needs a bit of rewording I think. It took me a minute to grasp what it is trying to convey.

Reply:
Alright. I gave it a rewrite with the aim of making it more clear and actionable.  I've also included a screen capture in the description so that you can see what it looks like with the fragment inlined.